src/sentry/integrations/github/integration.py

```diff
@@ -267,29 +268,48 @@ def dispatch(self, request: Request, pipeline: Pipeline) -> Response:
             pipeline.bind_state("reinstall_id", request.GET["reinstall_id"])
 
         if "installation_id" in request.GET:
+            organization = self.get_active_organization(request)
+
+            # We want to wait until the scheduled deletions finish or else the post install to migrate repos do not work.
+            integration_pending_deletion_exists = OrganizationIntegration.objects.filter(
+                integration__provider="github",
```

Comment:
We might have a provider enum somewhere

Reply:
I don't see an enum for provider in the Integration model. It's just a string. Ex usage: https://github.com/getsentry/sentry/blob/471eb0473085813a6bc6401abb4c5bcef4fdf785/src/sentry/integrations/pipeline.py#L79-L81
`integration_key` will come from the `GitHubIntegrationProvider`